pkg/controller/nodenetworkstate/nodenetworkstate_controller.go

```diff
@@ -36,8 +41,22 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	forThisPod := predicate.Funcs{
```

Comment:
forThisNode.

Do we care about Delete? There is nothing we can do, is it? It would probably fail on Update.

Reply:
Maybe just bypass them, since the only one delete state if node controller and the node is going down.